action/src/plugins/json/plugin.ts

```diff
@@ -27,19 +27,17 @@ const runJsonPlugin = async (
   const jsonString = fs.readFileSync(jsonToUpload).toString();
 
   if (jsonString.length > MAX_CHARACTERS) {
-    const message = `JSON string exceeds character limit. Limit: ${MAX_CHARACTERS}. Actual: ${jsonString.length}`;
-    core.error(message);
-    throw Error(message);
+    core.error(`JSON string exceeds character limit. Limit: ${MAX_CHARACTERS}. Actual: ${jsonString.length}. Skip`);
+    return;
   }
 
   let value;
 
   try {
     value = JSON.parse(jsonString);
   } catch (e) {
-    const message = `JSON file to upload does not have valid JSON contents: ${jsonToUpload}`;
-    core.error(message);
-    throw Error(message);
+    core.error(`JSON file to upload does not have valid JSON contents: ${jsonToUpload}. Skip.`);
+    return;
```

Comment:
So this makes the action not fail when reading bad data? Seems better than short circuiting everything.

Definitely out of scope here, but in the ideal case we should be able to communicate Stoat action failures to the comment directly.

Reply:
I think we can collect all the errors and warnings and surface them to the debug section.

There is already an issue for this: https://github.com/stoat-dev/stoat/issues/52

Will leave the error handling as is for this PR.